### PR TITLE
parser: add context information for unknown tags

### PR DESCRIPTION
**Pull request by Kijewski**

### CODE REVIEW

**Comment by Kijewski** [comment]
Please what the diff ignoring whitespaces changes, or it's entirely unreadable.

File: rinja_parser/src/node.rs
Comment (GuillaumeGomez) [inline rinja_parser/src/node.rs:209]: I can't wait for the `if let chain` feature to be stable...

**Review by GuillaumeGomez — DISMISSED** [review]

**Comment by GuillaumeGomez** [comment]
This is super cool, thanks! I'll need to re-approve once merge conflicts are fixed I think. ^^'

**Comment by Kijewski** [comment]
> This is super cool, thanks!

Thank you! :) Yes, oftentimes I was baffled myself why something does not parse correctly, so I hope this PR makes it clearer what is actually going on.

> I'll need to re-approve once merge conflicts are fixed I think. ^^'

Indeed :)

File: rinja_parser/src/node.rs
Comment (Kijewski) [reply rinja_parser/src/node.rs:209]: Yeah, I would like to remove a lot of `let Some(_) = f() else { return; }`*5 instances, too.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks again!

**Pull request merged**
